app/controllers/logs_controller.rb

```diff
@@ -371,8 +359,10 @@ def receipt
   end
 
   def export
-    start_date = Date.new(params[:start_date][:year].to_i, params[:start_date][:month].to_i, params[:start_date][:day].to_i)
-    stop_date = Date.new(params[:stop_date][:year].to_i, params[:stop_date][:month].to_i, params[:stop_date][:day].to_i)
+    parse_and_set_start_stop_dates(params)
+
+    start_date = Date.parse(params['start_date'])
+    stop_date = Date.parse(params['stop_date'])
```

Comment:
I think these can come out? And you'd update the `AND logs.when <= ?', start_date, stop_date)` below to use @start_date and @stop_date instead?